### PR TITLE
pinentry-bemenu: init at 0.9.0

### PR DESCRIPTION
**Pull request by joshua-cooper**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/security/pinentry-bemenu/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  src = fetchFromGitHub {
+    owner = "t-8ch";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-1hlPsPVkqVsx0yO2K1Sbq0A6FgE2Cra3LvfBsuqZjAM=";
+  };
+
+  nativeBuildInputs =
+    [ meson ninja pkg-config cmake libassuan libgpg-error popt bemenu ];
```
Comment (SuperSandro2000) [inline pkgs/tools/security/pinentry-bemenu/default.nix:16]: Please split nativeBuildInputs and buildInputs.

File: pkgs/tools/security/pinentry-bemenu/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "t-8ch";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-1hlPsPVkqVsx0yO2K1Sbq0A6FgE2Cra3LvfBsuqZjAM=";
+  };
+
+  nativeBuildInputs = [ meson ninja pkg-config cmake ];
```
Comment (SuperSandro2000) [inline pkgs/tools/security/pinentry-bemenu/default.nix:15]:
```
When evaluating attribute ‘pinentry-bemenu’:
warning: meson-cmake
Meson uses CMake as a fallback dependency resolution method and it likely is not necessary here. The message about cmake not being found is purely informational.

Near /home/hotpi/.cache/nixpkgs-review/pr-146613/nixpkgs/pkgs/tools/security/pinentry-bemenu/default.nix:15:3:
   |
15 |   nativeBuildInputs = [ meson ninja pkg-config cmake ];
   |   ^
See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/meson-cmake.md
```

Comment (joshua-cooper) [reply pkgs/tools/security/pinentry-bemenu/default.nix]: I've removed cmake now 👍

File: pkgs/tools/security/pinentry-bemenu/default.nix [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchFromGitHub, meson, ninja, pkg-config, libassuan
+, libgpg-error, popt, bemenu }:
+
+stdenv.mkDerivation rec {
+  pname = "pinentry-bemenu";
+  version = "0.6.0";
```
Comment (Ma27) [inline pkgs/tools/security/pinentry-bemenu/default.nix:6]: There's already a v0.7: https://github.com/t-8ch/pinentry-bemenu/releases/tag/v0.7.0

Comment (joshua-cooper) [reply pkgs/tools/security/pinentry-bemenu/default.nix]: Thanks! I've updated the version.

**Comment by Ma27** [comment]
LGTM, please resolve the conflict and we're good to go.

**Comment by joshua-cooper** [comment]
@Ma27 @SuperSandro2000 I've bumped the version up to the latest one. Can this be merged?

**Pull request merged**